packages/dicomImageLoader/src/imageLoader/isColorConversionRequired.ts

```diff
@@ -27,8 +27,7 @@ export default function isColorConversionRequired(imageFrame) {
   }
 
   if (photometricInterpretation === 'PALETTE COLOR') {
-    // since it is based on a look up table we don't need to convert
-    return false;
+    return true;
```

Comment:
This means we don't use any native LUT support @sedghi  - is there native LUT support for palette colour images?